region/maridia/outer/Mama Turtle Room.json

```diff
@@ -105,13 +112,23 @@
       "to": [
         {"id": 2},
         {"id": 3},
-        {"id": 5}
+        {"id": 5},
+        {"id": 6}
       ]
     },
     {
       "from": 5,
       "to": [
-        {"id": 2}
+        {"id": 2},
+        {"id": 6}
```

Comment:
Are there no imaginable strats that can get you from 1->5 but not 1->6?  Thinking weird stuff like -- gravity, double spring ball jump + pb boost but no walljump, or things yet to be discovered.  4->6 is different because you waka Mama Turtle.

Reply:
Actually I found one immediately.  If you have Gravity, High Jump, Speed Booster, Grapple, no Walljump, and no energy to Spark with, you can come in running at full speed and just speedy jump up to 5, but not 6.

Reply:
Also works swapping High Jump -> Spring Ball but much, much harder.

Reply:
Crossing from 5->6 requires grapple or space, but with that, you can just go 5->2->6. So we shouldnt need to add a 5->6 link since it is already included

Those strats you mentioned would be great to add

Reply:
5->2 with grapple is gated by Mama Turtle Asleep because it's tricky, but 5->6 doesn't care if she's asleep because it's an easy grapple.

Reply:
I dont see how these could be different difficulties - The only precise part of the strat is getting the grapple attached to the block

Reply:
Uhh.. maybe I'm bad?  I never miss the grapple itself.  What I always mess up and sometimes fall off is swinging up and collecting the item itself!  This is true whether I'm coming from 5 or 4 -> 2.

Reply:
> Those strats you mentioned would be great to add

It might look something like this:
```json
    {
      "link": [1, 5],
      "name": "Speedy Jump Assist",
      "entranceCondition": {
        "comeInRunning": {
          "speedBooster": true,
          "minTiles": 12
        }
      },
      "requires": [
        "Gravity",
        {"or": [
          {"and": [
            "canTrickyJump",
            "HiJump"
          ]},
          "canTrickySpringBallJump"
        ]}
      ]
    },
```

WDYT?  The `12` is totally made up we'd need to find the real number.

Reply:
I think the main concern is attaching to the grapple block. If you miss it you fall and die. You can actually collect the item without ever letting go of the grapple, so there should be no risk there

Reply:
> I think the main concern is attaching to the grapple block. If you miss it you fall and die. You can actually collect the item without ever letting go of the grapple, so there should be no risk there

Wait... what?  How you do this??

Reply:
This looked good, and I was trying to find the correct length, then I realized that this is probably harder than just riding Mama T, so I guess maybe it isnt important afterall.

Reply:
> This looked good, and I was trying to find the correct length, then I realized that this is probably harder than just riding Mama T, so I guess maybe it isnt important afterall.

I think it could matter in a low-energy situation where you want to avoid riding Mama T because you have like, no energy or need that energy to return (thus why you're not shinesparking in the first place).

Reply:
Riding Mama T damageless is in medium, and then crossing back without taking a hit should be fine with gravity (looks like its in basic)

Reply:
> > I think the main concern is attaching to the grapple block. If you miss it you fall and die. You can actually collect the item without ever letting go of the grapple, so there should be no risk there
> 
> Wait... what? How you do this??

Never mind, I was wrong. You can get to where youre stuck on top of the e-tank, and then let go with no risk, but its pretty annoying to get into that position.

Reply:
> Never mind, I was wrong. You can get to where youre stuck on top of the e-tank, and then let go with no risk, but its pretty annoying to get into that position.

Okay, whew, I thought I was crazy!  I've often collected the item with the grapple clipping as you describe, but then immediately fallen after item collection without being able to get on top of the grapple block itself, which would allow me to jump back left.